lib/api/apiUtils/object/createAndStoreObject.js

```diff
@@ -16,6 +16,9 @@ const validateWebsiteHeader = require('./websiteServing')
     .validateWebsiteHeader;
 const externalBackends = constants.externalBackends;
 
+const externalVersioningErrorMessage = 'For now, putting a version is ' +
+'not allowed on AWS or Azure backends';
```

Comment:
I would change this to: We do not currently support putting a versioned object to a location-constraint of type AWS or Azure.